be/src/olap/aggregate_func.h

```diff
@@ -203,6 +203,36 @@ struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_MIN, OLAP_FIELD_TYPE_LARGEINT>
     }
 };
 
+template <>
+struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_MIN, OLAP_FIELD_TYPE_VARCHAR> :
+        public AggregateFuncTraits<OLAP_FIELD_AGGREGATION_NONE, OLAP_FIELD_TYPE_VARCHAR> {
+    static void update(RowCursorCell* dst, const RowCursorCell& src, Arena* arena) {
+        bool src_null = src.is_null();
+        // ignore null value
+        if (src_null) return;
+
+        bool dst_null = dst->is_null();
+
+        Slice* dst_slice = reinterpret_cast<Slice*>(dst->mutable_cell_ptr());
+        const Slice* src_slice = reinterpret_cast<const Slice*>(src.cell_ptr());
+        if (dst_null || src_slice->compare(*dst_slice) < 0) {
```

Comment:
I this this if condition should be:

`if (!dst_null && src_slice->compare(*dst_slice) < 0)`

Did I misunderstand?

Reply:
Same here. Null always be replaced.